plutus-contract/src/Plutus/Contract/Test/Coverage.hs

```diff
@@ -48,13 +41,9 @@ getInvokedEndpoints es =
 getCoverageData :: [EmulatorEvent] -> CoverageData
 getCoverageData es =
   let extractLog e = case e of
```

Comment:
Does this need a TODO?

Reply:
Agree, seems like the same thing as with `scripts` command that I removed. I suppose we want to collect the executed scripts.